devel/appstream-glib/Portfile

```diff
@@ -44,7 +44,8 @@ depends_lib-append \
                     port:json-glib \
                     port:libarchive \
                     port:libxslt \
-                    port:libyaml
+                    port:libyaml \
+                    port:ossp-uuid
```

Comment:
Oh, you should rev-bump this, in order for the new dep to be installed on existing systems.

Reply:
Give me a moment